src/constants/adj_constants.opy

```diff
@@ -398,6 +398,8 @@ globalvar SETTING_ECHO_TANK_HP = createWorkshopSetting(bool, "General Changes",
 #!define ADJ_SOMBRA_SMG_DAMAGE 8
 #!define ADJ_SOMBRA_TRANSLOCATOR_COOLDOWN 6
 #!define ADJ_SOMBRA_VIRUS_COOLDOWN 6
+#!define ADJ_SOMBRA_STEALTH_REFUND_FRACTION createWorkshopSetting(int[0:100], "Sombra Stealth refund settings", "Percent of Stealth Timer Remaining Refunded Towards Translocator Cooldown", 40, 0) / 100
+#!define ADJ_SOMBRA_NEGATIVE_STATUS_PREVENTS_REFUND createWorkshopSetting(bool, "Sombra Stealth refund settings", "Negative Status Effects Cancelling Stealth Prevents Translocator Cooldown Refund", true, 1)
```

Comment:
No other workshop settings are defined in this file. I think the correct pattern would be the default value stored in constants, but the setting stored in the translocator file.

Reply:
I am tempted to just remove the settings all together but wanted an actual opinion besides my own regarding that. Seems to me the refund percentage should be up to toastie. But yeah defaulting to the codes default seems good.

Reply:
i have removed the workshop settings for now. they can always be pulled from the git history if we decide we need it for some reason.